docs/UsersGuide.asciidoc

```diff
@@ -307,44 +307,65 @@ image::images/review_badges.png[Review badges]
-
-Also github pull requests and issues can be linked using the generic format
-``<marker>[#<project/repo>]#<id>``, e.g. https://github.com/os-autoinst/openQA/issues/1234[gh#os-autoinst/openQA#1234], see https://github.com/os-autoinst/openQA/pull/973[gh#973]
-
-All issue references are stored within the internal database of openQA. The status can be updated using the `/bugs` API route for example using external tools.
+=== Bug references and labels
+==== Bug references
+It is possible to reference a bug by writing `<bugtracker_shortname>#<bug_nr>`
+in a comment, e.g. `bsc#1234`. It is also possible to spell out the full URL,
+e.g. `https://bugzilla.suse.com/show_bug.cgi?id=1234` which will then be
+shortened automatically. A bug reference is rendered as link and a bug icon
+is displayed for the job in various places as shown in the figure below.
+A comment containing a bug reference will also be
+<<UsersGuide.asciidoc#carry-over,carried over>> to reduce manual review work.
+
+WARNING: If you want to reference a bug without making it count as a bug
+reference you need to wrap it into a label (see subsequent section), e.g.
+`label:bsc#1234`.
+
+[[bug_icon]]
+.Bug icon for job with bug reference on test result overview
+image::images/bug_icon.png[Bug icon on test result overview]
+
+All bug references are stored within the internal database of openQA.
+The status can be updated using the `/bugs` API route with external tools.
+One can set the bug status this way which will then be shown in the web UI,
+see the figure below.
 
 [[labels_closed_tickets]]
-.Example for visualization of closed issue references. Upside down icons in red visualize closed issues.
+.Example for visualization of closed issue references. Upside down icons in red visualize closed progress issues.
 image::images/labels_closed_tickets.png[Example for visualization of closed issue references]
 
+NOTE: Also GitHub pull requests and issues can be linked. Use the generic format
+``<marker>[#<project/repo>]#<id>``, e.g.
+https://github.com/os-autoinst/openQA/issues/1234[`gh#os-autoinst/openQA#1234`].
+
+==== Labels
+A comment can also contain labels. Use `label:<keyword>` where `<keyword>`
+can be any valid character up to the next whitespace, e.g. `false_positive`.
+A label is rendered as yellow tag. The keywords are not defined within openQA
+itself. A valid list of keywords should be decided upon within each project
+or environment of one openQA instance. If a job has a label a special icon
+will be shown next to it in various places as shown in the figure below.
 
-=== Overwrite result of job ===
+[[label_icon]]
+.Label icon for job with a label on test result overview
+image::images/label_icon.png[Label icon on test result overview]
 
+NOTE: A label cannot contain a bug reference. If it does, it is nevertheless
```

Comment:
Sounds a bit weird. First "it cannot contain" and then it can actually contain but is not effective. Also with https://github.com/os-autoinst/openQA/pull/4998 you just improved the use of bugrefs in labels. How about

```suggestion
NOTE: A label containing a bug reference will still be treated as a label, not a bugref. The bugref will still be rendered as a link. That means no bug ...
```